keyboards/planck/keymaps/buhearns/keymap.c

```diff
@@ -0,0 +1,195 @@
+#include QMK_KEYBOARD_H
+#include "muse.h"
+
+extern keymap_config_t keymap_config;
+
+#define B_ENT FUNC(0) // Tap for enter, hold for right shift
```

Comment:
You can define this as `MT(MOD_RSFT, KC_ENT)` and remove the `fn_actions` section below (it is deprecated).

Reply:
Ah, thank you! That makes sense. I tried it with `LT(MOD_RSFT, KC_ENT)` and it obviously didn't work.